content/vault/v1.20.x/content/docs/deploy/kubernetes/vso/csi/index.mdx

```diff
@@ -0,0 +1,59 @@
+---
+layout: docs
+page_title: Direct secrets
+description: >-
+  Mount secrets from Vault directly into application containers with the Vault
+  Secrets Operator CSI driver.
+---
+
+# Direct secrets
+
+@include 'alerts/enterprise-only.mdx'
+
+The direct secrets feature is in **public beta**.
+
+While the Vault Secrets Operator lets you store Vault secrets in clusters as native
+Kubernetes Secrets, you can also use the "direct secrets" feature to mount Vault secrets directly to containers as Kubernetes [ephemeral volumes](https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/).
+
+The direct secrets feature installs the Vault Secrets
+Operator [CSI](https://github.com/container-storage-interface/spec/blob/master/spec.md) driver. 
+The CSI driver hooks into the Pod volume lifecycle, allowing applications to store secret data from Vault in ephemeral volumes with the same lifecycle as the application Pod itself.
+
+The Vault Secrets Operator CSI driver supports:
+- Syncing multiple secrets from multiple plugins. Supported plugins include:
+  - [Key/Value v1](/vault/docs/secrets/kv/kv-v1)
+  - [Key/Value v2](/vault/docs/secrets/kv/kv-v2)
+- Automatic generation and mounting of response-wrapped AppRole secret IDs as a
+  [trusted orchestrator](/vault/tutorials/app-integration/secure-introduction?in=vault%2Fapp-integration#trusted-orchestrator) for authentication between application Pods and Vault.
+- Helm-based [installation](/vault/docs/platform/k8s/vso/installation).
+- [Secret data transformation](/vault/docs/platform/k8s/vso/secret-transformation).
+
+The Vault Secrets Operator CSI driver does not support the [instant updates](/vault/docs/deploy/kubernetes/vso/sources/vault/instant-updates) feature.
```

Comment:
```suggestion
The Vault Secrets Operator CSI driver does not support [instant updates for `VaultStaticSecrets`](/vault/docs/deploy/kubernetes/vso/sources/vault/instant-updates).
```

Reply:
VSO-CSI doesn't use the custom resource called VaultStaticSecrets so I'm just going to put Instant Updates here with the link, I think